generalInfoPage/general.html

```diff
@@ -21,7 +21,32 @@
         </nav>
     </header>
 
+    <section class="info-section">
+      <img class="country-img" src="/img/Turkey.png" alt="">
+        
+        <div class="all-info">
+        <div class="info-div">
+          <div > <p class="country-info"><b>Turkey</b></p></div> 
+          <div class="row-info"> <i class="fas fa-money-bill-wave icons"></i><p class="P-icon"> Turkish-lira</p></div> 
+          <div class="row-info">  <i class="far fa-comment-dots icons"></i><p class="P-icon">Turkish</p></div>
+         <div class="row-info">  <i class="fas fa-phone-alt icons"></i><p class="P-icon">+90</p></div> 
+           <div class="row-info"><i class="fas fa-globe-americas icons"></i><p class="P-icon">Asia</p></div> 
+        </div>
+        <button class="btn-universities"><a class="btn-a" href="/Universities Page/Universities.html">Go to universities
+                list</a></button>
+</div>
+    </section>
```

Comment:
a lot of white spacing, delete them,please

Reply:
done